doc/source/toolchain.rst

```diff
@@ -71,9 +71,10 @@ The table shows the NumPy versions suitable for each major Python version
 =================  ========================    ===========================
 2.7 (SciPy 1.2)      1.8.2                      1.16.x
 3.5 (SciPy 1.4)      1.13.3                     1.18.x
-3.6                  1.14.5                     >= 1.18.x
-3.7                  1.14.5                     >= 1.18.x
+3.6 (SciPy 1.5)      1.14.5                     >= 1.18.x
```

Comment:
```suggestion
3.6 (SciPy 1.5)      1.14.5                     1.19.x
```

Reply:
@rgommers You ok with this modification?

Reply:
thanks, adopted